dev-docs/update-deno_jsonc.md

```diff
@@ -0,0 +1,10 @@
+## tl;dr
+
+deno.jsonc currently doesn't support globs, so we have a script to expand the globs ourselves.
+
+## Steps
+
+```
+$ cd quarto-cli
+$ deno run --allow-all package/src/common/create-deno-config.ts > deno.jsonc
```

Comment:
@cscheid is the ts file is missing ? I don't see it in the PR and not in my local clone of main